var/spack/repos/builtin/packages/py-pennylane/package.py

```diff
@@ -12,15 +12,24 @@ class PyPennylane(PythonPackage):
 
     homepage = "https://docs.pennylane.ai/"
```

Comment:
Same here as with Py-PennyLane-Lightning. Do we need to change this file?

Reply:
No strictly necessary. I thought I would just get all `py-pennylane` packages in-line as possible.